src/com/lilithsthrone/game/dialogue/utils/PhoneDialogue.java

```diff
@@ -3072,75 +3072,66 @@ public Response getResponse(int responseTab, int index) {
-								+ "</p>");
-						Main.game.endTurn(60*15);
-						Main.game.setContent(new Response("", "", Main.game.getDefaultDialogue()));
+						Loiter(15);
 					}
 				};
 				
 			} else if(index == 2) {
 				return new ResponseEffectsOnly("1 hour", "Loiter in this area for the next hour.") {
 					@Override
 					public void effects() {
-//						Main.mainController.openPhone();
-						Main.game.getTextStartStringBuilder().append(
-								"<p style='text-align:center;'>"
-										+ "<i>You spend the next hour loitering about, doing nothing in particular...</i>"
-								+ "</p>");
-						Main.game.endTurn(60*60);
-						Main.game.setContent(new Response("", "", Main.game.getDefaultDialogue()));
+						Loiter(60);
 					}
 				};
 				
 			} else if(index == 3) {
 				return new ResponseEffectsOnly("4 hours", "Loiter in this area for the next four hours.") {
 					@Override
 					public void effects() {
-//						Main.mainController.openPhone();
-						Main.game.getTextStartStringBuilder().append(
-								"<p style='text-align:center;'>"
-										+ "<i>You spend the next four hours loitering about, doing nothing in particular...</i>"
-								+ "</p>");
-						Main.game.endTurn(60*60*4);
-						Main.game.setContent(new Response("", "", Main.game.getDefaultDialogue()));
+						Loiter(60*4);
 					}
 				};
 				
 			} else if(index == 4) {
 				return new ResponseEffectsOnly("8 hours", "Loiter in this area for the next eight hours.") {
 					@Override
 					public void effects() {
-//						Main.mainController.openPhone();
-						Main.game.getTextStartStringBuilder().append(
-								"<p style='text-align:center;'>"
-										+ "<i>You spend the next eight hours loitering about, doing nothing in particular...</i>"
-								+ "</p>");
-						Main.game.endTurn(60*60*8);
-						Main.game.setContent(new Response("", "", Main.game.getDefaultDialogue()));
+						Loiter(60*8);
 					}
 				};
 				
 			} else if(index == 5) {
 				return new ResponseEffectsOnly("12 hours", "Loiter in this area for the next twelve hours.") {
 					@Override
 					public void effects() {
-//						Main.mainController.openPhone();
-						Main.game.getTextStartStringBuilder().append(
-								"<p style='text-align:center;'>"
-										+ "<i>You spend the next twelve hours loitering about, doing nothing in particular...</i>"
-								+ "</p>");
-						Main.game.endTurn(60*60*12);
-						Main.game.setContent(new Response("", "", Main.game.getDefaultDialogue()));
+						Loiter(60*12);
 					}
 				};
 			}
 			return null;
 		}
 
+		private void Loiter(int minutes) {
+			String period = "period";
+			if (minutes == 15) {
+				period = "fifteen minutes";
+			} else if (minutes == 60) {
+				period = "hour";
+			} else if (minutes == 60 * 4) {
+				period = "four hours";
+			} else if (minutes == 60 * 8) {
+				period = "eight hours";
+			} else if (minutes == 60 * 12) {
+				period = "twelve hours";
+			}
```

Comment:
This should probably be decoupled from the caller's values by using `Util.intToString(int)`. Something like this should work (don't worry about counting spaces... the eventual html interpretation will combine them):

```
String period = ""
int hours = minutes / 60;
int partialMinutes = minutes % 60;
if (hours != 0) {
    period += Util.intToString(hours) + " hours ";
}
if (partialMinutes != 0) {
    period += Util.intToString(partialMinutes) + " minutes "
}
if (minutes == 0) {
    period = "period "
}
```

Reply:
Done, including check to generate "You spend the next hour" instead of "You spend the next one hours".